app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
+                 clean_power,
+                 average_rating: float,
+                 count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list):
+        income = []
+        for car in cars:
+            income.append(self.wash_single_car(car))
+        return sum(income)
```

Comment:
in this method, you should call `wash_single_car` method

Reply:
why is this doesn't count as calling `wash_single_car` method? It is in the method

Reply:
hm, okay, you just replace logic, got it. Fix type annotation